client/src/app/site/mediafiles/components/mediafile-list/mediafile-list.component.ts

```diff
@@ -397,7 +386,8 @@ export class MediafileListComponent extends BaseListViewComponent<ViewMediafile>
      * @returns the display name of the selected action
      */
     public getNameOfAction(mediaFileAction: string): string {
```

Comment:
A quick request: Can you find another name than `action`. It raises confusion with our new concepts.. E.g. logos and fonts can have usages/places where they are used. You'll find something appropriate I think :)